flutter-for-android.md

```diff
@@ -0,0 +1,1845 @@
+---
+layout: page
+title: Flutter for Android Developers
+permalink: /flutter-for-android/
+---
+This document is meant for Android developers looking to transfer their existing Android knowledge to build mobile apps in Flutter. If you understand the fundamentals of the Android framework then you can use this document as a jump start to Flutter development.
+
+This document can be used as a cookbook by jumping around and finding questions that are most relevant to your needs.
+
+# Views
+
+**What is the equivalent of a View in Flutter?**
```

Comment:
Global: please make these start with `##` and remove the `**`

Reply:
Done